js/gamestate.js

```diff
@@ -198,6 +196,9 @@ class GameState {
 
   draw() {
     // Make a copy so that we can edit this card
+    if (!this.rng) {
```

Comment:
I get an error otherwise

Andrew Johnson
Founder
*gaiagps.com <http://gaiagps.com>*

On Sat, Jun 13, 2015 at 7:08 AM, Kevin Lacker <notifications@github.com>
wrote:

> In js/gamestate.js
> <https://github.com/lacker/cardkit/pull/14#discussion_r32370732>:
>
> > @@ -198,6 +196,9 @@ class GameState {
> >
> >    draw() {
> >      // Make a copy so that we can edit this card
> > +    if (!this.rng) {
>
> I don't think you should create a new rng here. It should be in game-start
> if anything
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lacker/cardkit/pull/14/files#r32370732>.
>